graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/GroupBy/FieldComponent.tsx

```diff
@@ -38,13 +38,16 @@ const FieldComponent = ({ index, fieldType }: Props) => {
 
     if (fieldType !== newFieldType) {
       if (newFieldType === 'time') {
+        setFieldValue(`groupBy.groupings.${index}.limit`, undefined, false);
+
         setFieldValue(`groupBy.groupings.${index}.interval`, {
```

Comment:
Maybe we are also able to do:
```suggestion
        setFieldValue(`groupBy.groupings.${index}`, { interval: {
```